<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatWindow</class>
 <widget class="QMainWindow" name="chatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="chatBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>511</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="chatEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>430</y>
      <width>511</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="friendMessage">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>270</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;这里待定，显示信息&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="friendIcon">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>70</y>
      <width>141</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>这里是对方头像</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendMessage">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>430</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>490</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fileSend">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>文件</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
